src/common/localization/en-us.ts

```diff
@@ -322,6 +322,69 @@ export const english: IAppStrings = {
     profile: {
         settings: "Profile Settings",
     },
+    shortcuts: {
+        squareBrackets: {
+            name: "square brackets",
+            ariaLabel: "square brackets",
+            keys: {
+                key1: "[",
+                key2: "]",
+            },
+            description: "Can be used move the selection to the previous or the next word",
+        },
+        greaterAndLessThan: {
+            name: "greater-than and less-than",
+            ariaLabel: ",  or  .",
+            keys: {
+                key1: ",",
+                key2: ",",
+                },
+            description: "Can be used go to the previous or the next page in multi-pages documents",
```

Comment:
Suggestion: remove "can be use to,"  similar to 
![image](https://user-images.githubusercontent.com/60453211/82237510-9b178300-98ea-11ea-808b-90d92ce52fba.png)

Reply:
Yeah, It feels redundant  - I took it from official readme, though. Let's check with Kun - it's easy change.

Reply:
@kunzms

Reply:
Yes, remove "Can be used "